diff --git a/build.rs b/build.rs
index 0233ade..5288d92 100644
--- a/build.rs
+++ b/build.rs
@@ -1,5 +1,5 @@
 extern crate lalrpop;
-use std::process::Command;
+use std::process::{Command, Stdio};
 
 fn main() {
     lalrpop::Configuration::new()
@@ -15,4 +15,8 @@ fn main() {
         .unwrap();
     let git_hash = String::from_utf8(output.stdout).unwrap();
     println!("cargo:rustc-env=GIT_HASH={}", git_hash);
+    let cmd = String::from_utf8(Command::new("llvm-config").arg("--libs").stderr(Stdio::inherit()).output().unwrap().stdout).unwrap();
+    for lib in cmd.split_whitespace() {
+        println!("cargo:rustc-link-lib={}", &lib[2..]);
+    }
 }
